scripts/update_and_rebuild_petsc.sh

```diff
@@ -96,10 +102,8 @@ if [ -z "$go_fast" ]; then
       --F77FLAGS='-fPIC -fopenmp' \
       --with-cxx-dialect=C++11 \
       --with-fortran-bindings=0 \
-      --with-sowing=0 \
-      $* \
```

Comment:
This allows us to append options from command lines. Why removed  it?

Reply:
I'll put it back.